stories/Tokens/utils/functions.tsx

```diff
@@ -0,0 +1,68 @@
+import { copyButton } from './CopyButton';
+import { type TokenGenericJsonObject } from './Tokens.types';
+import { Table } from '../../../packages/dds-components/src';
+
+export function splitReferenceKeys(v: string): Array<string> {
+  return v.replace(/\{|\}/g, '').split('.');
+}
+
+export function isReferencedValue(v: string): boolean {
+  return v[0] === '{' && v[v.length - 1] === '}';
```

Comment:
Oj 😅 virker litt hacky, men er kanskje greit?

Reply:
Har ikke en bedre ide 😅 alle referanser bruker krøllparenteser, men i teorien kan en verdi bestå av kombinasjon av referanse og hardkodet verdi eller flere referanser. Har du noen forslag? 🤔

Reply:
Tja nei. Sikkert greit dette 🤷🏻 Kjenner ikke egentlig godt nok til oppsettet rundt referanser og sånt kjenner jeg.

Kan evt. gjøre noe sånn for litt bedre lesbarhet:
```suggestion
  return v.startsWith('{') && v.endsWith('}')
```
Men kanskje subjektivt 😅

Reply:
Enig i forslaget, mer oversiktlig. Da merger jeg 👍